src/sentry/utils/env.py

```diff
@@ -1,5 +1,16 @@
 import sys
 
+from django.conf import settings
 
-def in_test_environment():
+
+def in_test_environment() -> bool:
     return "pytest" in sys.argv[0] or "vscode" in sys.argv[0]
+
+
+def is_split_db() -> bool:
+    if len(settings.DATABASES) != 1:  # type: ignore
+        return True
+    for db in settings.DATABASES.values():  # type: ignore
+        if db["NAME"] in {"region", "control"}:
+            return True
```

Comment:
When the application runs in split silo mode, neither control or region connections exist. Having this available in a generic location like this could be a footgun for future us.